hpx/parallel/execution_policy.hpp

```diff
@@ -1552,6 +1552,9 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
           : inner_(std::move(policy.inner_))
         {}
 
```

Comment:
@dcbdan @hkaiser  Is this a valid explanation for the copy constructor? Added some simple wording so it would be documented